Objects and Functions.js

```diff
@@ -27,4 +27,39 @@ if ( checkName () ) {
     alert( 'Доступ получен' );
 } else {
     alert( 'Доступ закрыт' );
-}
\ No newline at end of file
+}
+
+
+
+
+
+
+
+let item = "Car";

```

Comment:
This is wrong, you need to create an object, add an attribute to it, then add another attribute to this object. You initially create strings and try to do something with it. Read https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object